migrate.js

```diff
@@ -32,29 +32,41 @@ function makeFileExistsObv(filename) {
 }
 
 function getOldLogStreams(sbot, config) {
-  if (sbot.createRawLogStream && sbot.createSequenceStream) {
+  if (sbot.createRawLogStream && sbot.status) {
     const logStream = sbot.createRawLogStream({ old: true, live: false })
     const logStreamLive = sbot.createRawLogStream({ old: false, live: true })
-    const sizeStream = pull(
-      sbot.createSequenceStream(),
-      pull.filter((x) => x >= 0)
-    )
-    return [logStream, logStreamLive, sizeStream]
+    const getSize = () => sbot.status().sync.since
+    return [logStream, logStreamLive, getSize]
+  }
+  if (sbot.createLogStream && sbot.status) {
+    const logStream = sbot.createLogStream({
+      raw: true,
+      old: true,
+      live: false,
+    })
+    const logStreamLive = sbot.createLogStream({
+      raw: true,
+      old: false,
+      live: true,
+    })
+    const getSize = () => sbot.status().sync.since
```

Comment:
Should this be since.value instead?

Reply:
There same a few lines up

Reply:
It's weird, isn't it? But it's correct,

- see 20.3.0: https://github.com/ssbc/ssb-db/blob/da89b7552e054c9e8baaad78506714e13a62a7ec/index.js#L107
- see 19.3.1: https://github.com/ssbc/ssb-db/blob/6c0cdd580b6166b106283e3f1745585e189252ce/index.js#L94

Reply:
OMG!